### DISCUSSION TITLE
Invalid account number

### DISCUSSION THREAD

**Discussion by Milyanetun** [Authenticator-Extension/Authenticator#653] [Q&A]
Hi!
In my work, we use this app by default. Unfortunately, I had to re-install Windows, and, therefore, I had to re-install the plugin. Although the data I insert is correct (and I checked it with our internal support team), it keeps telling me that I have an "invalid account number". I have no idea how to fix it. In my company, they said they could not affect it anyhow. 
I thought that it might be because I am currently in Brazil, while my computer had the timezone of my home country (Russia). I corrected it but nothing helped. Someone, please, help me.

**Comment by mymindstorm** [comment]
Have you tried menu -> sync clock with Google? Try turning on network time in windows settings.

**Reply by Milyanetun** [reply]
Yes, I did both. It did not help.

**Reply by mymindstorm** [reply]
Try using the QR button at the top right of the account and importing the entry to another device. Does that device generate correct codes?

**Reply by Milyanetun** [reply]
I have installed the version for ipad, and tried to use the generated QR. It did not work because I cannot enter my account so the QR is also invalid.

**Comment by mymindstorm** [comment]
You should download an app like Google Authenticator or Microsoft Authenticator which will allow you to scan a QR code to add an account.
-------- Original message --------From: Milyanetun ***@***.***> Date: 4/29/21  12:32 PM  (GMT-06:00) To: Authenticator-Extension/Authenticator ***@***.***> Cc: mymindstorm ***@***.***>, Comment ***@***.***> Subject: Re: [Authenticator-Extension/Authenticator] Invalid account number (#653) 
I am sorry I am not sure how to do it. I press the QR button, the code appears but I do not understand how to send it to another device.

**Reply by Milyanetun** [reply]
![image](https://user-images.githubusercontent.com/56788332/116596097-b1b4bf80-a92c-11eb-89aa-91874281bbdc.png)


It says that the token is invalid.

**Comment by Sneezry** [comment]
![C73AE518-B401-44F8-AD89-F40DD9CF8596](https://user-images.githubusercontent.com/1621293/116598035-f1919c00-a958-11eb-8868-16985d3d158e.jpeg)

Milyanetun ***@***.***>于2021年4月30日 周五上午1:52写道：

**Reply by Milyanetun** [reply]
I cannot enter my account so it looks like that. My problem is that I cannot sign in to plugin because it says "invalid account number".
![image](https://user-images.githubusercontent.com/56788332/116601203-e6c41080-a932-11eb-92fb-61e5c2e0c051.png)

**Comment by Sneezry** [comment]
> Although the data I insert is correct (and I checked it with our internal support team), it keeps telling me that I have an "invalid account number".

Could you provide more information for:
1. What is the data, is it your 2fa secret?
2. How did you insert the data? Did you add a new account by scanning a QR image or by entering a secret manually?
3. Where did you see "invalid account number"? Is it the Authenticator extension or the website you tried to login?

**Reply by Milyanetun** [reply]
Hi!

1. I received the data from my work. It is the secret and the Username.
2. Manually.
3. It is the Authenticator. Here is what it looks like more or less (I am sorry it is in Russian, I could not find where to change the language for you). I write the data (it is the same data I used 9 months ago to start Authenticator and the same I used for the internal mobile app, so I am sure it is correct) in the Authenticator, press "ready", and it says "Invalid Account Number".

![image](https://user-images.githubusercontent.com/56788332/116696210-46b9c600-a9ca-11eb-840e-60688873785a.png)

**Comment by Sneezry** [comment]
The extension will show invalid account secret when:

1. the secret is shorter than 16
2. the secret is neither base32 nor hex string

base32 string: only contains a-z, 2-7 and =
hex string: only contains a-f, 0-9

You may check your secret to figure out why the extension shows this notification. If the reason is the secret is too short, please let me know, and I will consider to change this limit.
